src/util/secp256k1.rs

```diff
@@ -426,6 +430,28 @@ mod tests {
         result: R
     }
 
+    #[test]
+    fn test_parse_serialize_compressed() {
+        let mut t1 = Secp256k1PrivateKey::new();
+        t1.set_compress_public(true);
+        let h_comp = t1.to_hex();
+        t1.set_compress_public(false);
+        let h_uncomp = t1.to_hex();
+
+        assert!(&h_comp != &h_uncomp);
+        assert_eq!(h_comp.len(), 66);
+        assert_eq!(h_uncomp.len(), 64);
```

Comment:
Can you also test that the final two characters in `h_comp` are `01`, and that `h_comp` and `h_uncomp` are equal up to the 64th byte?

Reply:
Change coming from #1203 - addressed with https://github.com/blockstack/blockstack-core/pull/1202/commits/cbdd3acefeae69cc66e215aaa9d56732dbefa184

Reply:
Ah, gotcha.  Thanks for fixing it nevertheless :+1: